https://github.com/trilinos/Trilinos/pull/5559

commit 26e978bbeb86cee8e83cf80610ed2fadecc7b6c0
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Mon Jul 22 13:00:24 2019 +0200

    Sort input file list
    
    so that variables_f.js builds in a reproducible way
    in spite of indeterministic filesystem readdir order.
    
    See https://reproducible-builds.org/ for why this is good.

diff --git a/doc/build_docs.pl b/doc/build_docs.pl
index 967ec8ac04..3efb64e550 100755
--- a/doc/build_docs.pl
+++ b/doc/build_docs.pl
@@ -37,7 +37,7 @@ use File::Basename;
 print "\n";
 if(!$skipBuild) {
   print "\nGenerating package documentation ...\n";
-  find (\&buildDocs, $trilinosDir);
+  find ({wanted=>\&buildDocs, preprocess => sub {sort {$a cmp $b} @_}}, $trilinosDir);
   if($build_dir) {
     find (\&copyXML, $trilinosDir);
   }
